test/letter-avatars.ts

```diff
@@ -24,21 +25,19 @@ describe('generateAvatarURL() gravatar enabled', function () {
   })
 
   it('should return correct urls for names with spaces', function () {
-    assert.strictEqual(avatars.generateAvatarURL('Daan Sprenkels'), 'http://localhost:3000/user/Daan%20Sprenkels/avatar.svg')
+    assert.strictEqual(avatars.generateAvatarURL('Daan Sprenkels'), 'https://cdn.libravatar.org/avatar/4c274a80fba9203139aa2908da7832a5?s=400')
```

Comment:
Was this test changed intentionally?

Reply:
Good catch! I think I was confused by the failing test, but it turns out it should fail 😅.
@DerMolly seems to have inadvertently changed the behaviour of `generateAvatarURL` with the TS-migration. I will do a PR to fix that shortly.